solution/ui/regulations/eregs-vite/src/views/Resources.vue

```diff
@@ -197,6 +210,17 @@ export default {
                 },
             });
         },
+        async synonymLinks(synonym){
```

Comment:
Is this needed?  won't it just happen for free when you re-execute search?

Reply:
It is but I refactored to use most of execute searches code